modules/ROOT/pages/addition/docker/container.adoc

```diff
@@ -67,8 +111,57 @@ NEO4J_server_metrics_prometheus_endpoint="localhost:2004" \
 NEO4J_server_metrics_filter="*" \
 agent console -s
 ----
+[source, shell, role=noheader]
+----
+ docker run --entrypoint /custom/entrypoint.sh -e NEO4J_ACCEPT_LICENSE_AGREEMENT=yes -v <path/to/custom/entrypoint>:/custom $NEO4J_IMAGE
+----
+
+* In service mode
 +
+.entrypoint.sh
+[source, shell]
+----
+#!/bin/bash -eu
+
+/startup/docker-entrypoint.sh neo4j &
+agent service install
+cat > neo4j-ops-manager-agent.service <<EOF
+[Service]
+Environment="CONFIG_SERVER_ADDRESS=<server grpc address>"
+Environment="CONFIG_TOKEN_URL=<server http login url>"
+Environment="CONFIG_TOKEN_CLIENT_ID=<client id>"
+Environment="CONFIG_TOKEN_CLIENT_SECRET=<client secret>"
+Environment="CONFIG_TLS_TRUSTED_CERTS=</path/to/trusted/certs/pem/file>"
+Environment="CONFIG_LOG_FILE=</path/to/nom-agent/log.txt>"
+Environment="CONFIG_INSTANCE_1_NAME=<instance name>"
+Environment="CONFIG_INSTANCE_1_BOLT_URI=<bolt uri of the local instance>"
+Environment="CONFIG_INSTANCE_1_BOLT_USERNAME=<local instance user name>"
+Environment="CONFIG_INSTANCE_1_BOLT_PASSWORD=<local instance password>"
+Environment="CONFIG_INSTANCE_1_QUERY_LOG_PORT=<an available port>"
+Environment="CONFIG_INSTANCE_1_LOG_CONFIG_PATH=<path to server-logs.xml>"
+EOF
+systemctl start neo4j-ops-manager-agent.service
+----
 [source, shell, role=noheader]
 ----
  docker run --entrypoint /custom/entrypoint.sh -e NEO4J_ACCEPT_LICENSE_AGREEMENT=yes -v <path/to/custom/entrypoint>:/custom $NEO4J_IMAGE
 ----
+
+== External Monitoring
+
+NOM agent can be run in either the console or service mode on the Docker host and configured to have access to Neo4j container resources. 
+Following additional configurations to be applied to Neo4j container run config to enable external NOM agent to monitor the instance correctly:
+
+[source, shell]
+----
+docker run --entrypoint /custom/entrypoint.sh -e NEO4J_ACCEPT_LICENSE_AGREEMENT=yes \
+           -v <path/to/custom/entrypoint>:/custom \
+           -v </path/to/local/neo4j/home>:/var/lib/neo4j \ <1>
+           -p "8884:2004" \ <2>
+           -p "9500:9500" \ <3>
+       neo4j/neo4j:latest
+----
+
+<1> Neo4j home directory needs to be mounted back on the Docker host to enable access to agent.
```

Comment:
```suggestion
Note that:
. The Neo4j home directory needs to be mounted back on the Docker host to enable access to agent.
```

Reply:
I think `callouts` make more sense in this section than a note. WDYT @AlexicaWright ?

Reply:
@ravikiran-sulikeri Ravi, fyi, there is currently an [issue](https://github.com/neo4j-documentation/docs-ui/pull/204) with callouts but I added them in anticipation that it gets fixed.

Reply:
Waiting for word on whether it's getting fixed in the near future. If so, I'd say keep the annotation.